src/main/java/net/snowflake/ingest/streaming/internal/AbstractRowBuffer.java

```diff
@@ -442,6 +442,26 @@ Set<String> verifyInputColumns(
               "Values for all non-nullable columns must be specified, rowIndex:%d", rowIndex));
     }
 
+    // Check for null values in not-nullable columns in the row
+    List<String> nullValueNotNullCols = new ArrayList<>();
```

Comment:
Does it make sense to differentiate between `missingCols` and `nullValueNotNullCols`?
Is there a conceptual difference between the cases when a column is present with `null` value and when it isn't present at all.

Reply:
i was thinking about this, the only thing is the naming of missingCols. But if we do like this than we don't need to adapt anything in kafka push code. What u think @sfc-gh-tzhang ?

Reply:
I'm fine with combing it and we will need to update the name to make it clear, the only concern is that we're updating a public API, but I highly doubt any customer is using this.